ES Options Cleanup 2 + Updates

Darknior opened the pull request:
DOLPHIN

- Add Wii and GC rom path
- Remove "internal_resolution" old option and replace by the new es_features system with real values

PCSX2

- Remove old "fullboot" and replace by the new es_features system
- Remove "internal_resolution" old option and replace by the new es_features system with real values
- Add "games cheat" option
- Add "widescreen patches" option
- Add "automatic gamefixes" option
- Add "manual gemefixes" with "VuAddSubHack" hack for @liberodark

PPSSPP

- Remove "internal_resolution" old option and replace by the new es_features system with real values
- Add "Anisotropic filtering" option
- Add "Texture scalling level" and "type" option
- Add "Texture Deposterize" option

I have try to add "Vulkan" backend, but it not works for now

WII

- Remove "emulated_wiimotes" old option and replace by the new es_features system with real values

CEMU

- Remove "emulated_wiimotes" old option and replace by the new es_features system with real values
- Add Async shader compilation option
- Add the default "ROM path" directory
- Update the "Game Language" default to the system one
- Add "fps info" in debug mode

CITRA

- Remove "screen_layout" old option and replace by the new es_features system with real values

Now we can remove old functions from Emulationstation
https://github.com/batocera-linux/batocera-emulationstation/blob/master/es-app/src/guis/GuiMenu.cpp#L3990

nadenislamarre commented:
- CemuGenerator.setSectionConfig(config, xml_root, "console_language", str(getGameCubeLangFromEnvironment())) ?!?
- please don't rewrite comments (for future pr, not this one) with alignment and uppercase. this is custom preference and it makes pr unreadable. "# Add Default Sections" => why to add a space and an uppercase (and same for most of the pr) ? next one will come and remove the uppercase because he prefers the lowercase. Rule : don't modify comment unless you just want to add content. This is true for indentation too. I'm unable to find what's important in your pr.it was harder to find : getGameCubeLangFromEnvironment which is for sure wrong.

Darknior commented:
Fixed a mistake on FPS debug option

`CemuGenerator.setSectionConfig(config, xml_root, "console_language", str(getGameCubeLangFromEnvironment())) ?!?`

Yes sorry i have modified the function with the good CEMU id list, not the DOLPHIN on, but i missed to rename it.
Now i have renamed it and it works fine. I have try again with system in french games are in french, with system in english games are in english

Dolphin :
`availableLanguages = { "en_US": 0, "de_DE": 1, "fr_FR": 2, "es_ES": 3, "it_IT": 4, "nl_NL": 5 }`

Cemu :
`availableLanguages = { "ja_JP": 0, "en_US": 1, "fr_FR": 2, "de_DE": 3, "it_IT": 4, "es_ES": 5, "zh_CN": 6, "ko_KR": 7, "hu_HU": 8, "pt_PT": 9, "ru_RU": 10, "zh_TW": 11 }
`
And i've not missed : `from os import environ`
